wgsl/index.bs

```diff
@@ -7259,6 +7260,7 @@ value with the same sign.
   <tr><td>`modf(x)`<td>Correctly rounded
   <tr><td>`normalize(x)`<td>Inherited from `x / length(x)`
   <tr><td>`pow(x, y)`<td>Inherited from `exp2(y * log2(x))`
+  <tr><td>`radians(x)`<td>Inherited from `x * 0.017453292519943295474`
```

Comment:
there is a little bit of awkwardness in that, essentially, all the functions working with angles receive them in radians. Except for this one.

Reply:
But that's the whole point of the function.

Reply:
But that's the whole point of the function.